File: MODUL 3/PRAK302-2310817210021-RAKHMAD FAUZHAN RAMADHAN.c
#include <stdio.h>

char konversi_nilai(int nilai) {
    if (nilai >= 80) {
        return 'A';
    } else if (nilai >= 70 && nilai <= 79) {
        return 'B';
    } else if (nilai >= 60 && nilai <= 69) {
        return 'C';
    } else if (nilai >= 50 && nilai <= 59) {
        return 'D';
    } else {
        return 'E';
    }
}

int main() {
    int nilai;
    for (int i = 0; i < 5; i++) {
        scanf("%d", &nilai);
        printf("%c\n", konversi_nilai(nilai));
    }
    return 0;
}

File: MODUL 3/PRAK302-2310817210021-RAKHMAD FAUZHAN RAMADHAN.py
def konversi_nilai(nilai):
    if nilai >= 80:
        return "A"
    elif 70 <= nilai <= 79:
        return "B"
    elif 60 <= nilai <= 69:
        return "C"
    elif 50 <= nilai <= 59:
        return "D"
    else:
        return "E"

nilai_list = []
for _ in range(5):
    nilai = int(input())
    nilai_list.append(nilai)

for nilai in nilai_list:
    print(konversi_nilai(nilai))
